### PR TITLE
Bugfix staking

### PR DESCRIPTION
**Pull request by dmitrii602**
# [VLWA-981](https://velasnetwork.atlassian.net/browse/VLWA-981)

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Step 1
- [ ] Step 2

### Platforms tested on

- [ ] Android
- [ ] IOS

### CODE REVIEW

File: components/TabsBarValidator.js [hunk 1]
```diff
@@ -1,277 +1,356 @@
...
+              console.error(err);
+              return;
+            }
+            changePage("confirmWithdrawal")();
+          });
+        };
 
-      }
+        const copyAddress = async () => {
+          const DURATION = 1000 / 10;
```
Comment (FrozenPyrozen) [inline components/TabsBarValidator.js:114]: Maybe separate `const DURATION = 1000 / 10;` into upper level?

File: web3t/providers/insight.js [hunk 2]
```diff
@@ -852,4 +863,4 @@
     };
     return _curry();
   }
-}).call(this);
+}).call(this);
```
Comment (FrozenPyrozen) [inline web3t/providers/insight.js:866]: Not critical, but recomended)
image

**Review by FrozenPyrozen — CHANGES_REQUESTED** [review]

File: components/ButtonBlock.js [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+import { StyleSheet, Text, Platform, Vibration, Alert } from "react-native";
+import { Button, Toast } from "native-base";
 import Images from "../Images";
+import { useNetInfo } from "@react-native-community/netinfo";
 
 export default (props) => {
+  const netInfo = useNetInfo();
+  const validatorNet =
+    !netInfo.details ||
+    netInfo.isConnected === true ||
```
[the remark this answers is not in the document]
Comment (dmitrii602) [reply components/ButtonBlock.js]: done

File: components/TabsBarValidator.js (hunk 1 above)
Comment (dmitrii602) [reply components/TabsBarValidator.js]: done

File: web3t/providers/insight.js (hunk 2 above)
Comment (dmitrii602) [reply web3t/providers/insight.js]: done

**Review by FrozenPyrozen — DISMISSED** [review]

File: components/ButtonBlock.js [hunk 4]
```diff
@@ -38,12 +38,12 @@ export default (props) => {
...
     switch (type) {
-      case "REQUEST_WITHDRAW":
-        return { color: "#fff" };
-      case "WITHDRAW":
-        return { color: "#fff" };
-      case "DISABLED":
-        return { color: "gray" };
+      case 'REQUEST_WITHDRAW':
+        return { color: '#fff' };
+      case 'WITHDRAW':
```
Comment (FrozenPyrozen) [inline components/ButtonBlock.js:43]: We can use 
```
case 'REQUEST_WITHDRAW':
case 'WITHDRAW':
           return { color: '#fff' };
```

**Review by FrozenPyrozen — CHANGES_REQUESTED** [review]

**Review by FrozenPyrozen — APPROVED** [review]

**Review by FrozenPyrozen — APPROVED** [review]

**Review by Corresponsible — APPROVED** [review]

**Pull request merged**